### PR TITLE
Remove Capacity and Top Consumers card

### PR DESCRIPTION
**Pull request by rawagner**

### CODE REVIEW

File: frontend/public/components/utils/units.js [hunk 1]
```diff
@@ -382,3 +382,8 @@ export const convertToBaseValue = (value) => {
   // Unrecognized unit.
   return null;
 };
+
+export const secondsToNanoSeconds = (value) => {
+  const val = Number(value);
+  return Number.isFinite(val) ? val * 1000 ** 3 : 0;
```
Comment (vojtechszocs) [inline frontend/public/components/utils/units.js]: This is the [exponentiation operator](https://github.com/tc39/proposal-exponentiation-operator) which is semantically equivalent to `Math.pow`.

Nit: I'd put the expression `1000 ** 3` in braces to improve readability.

Comment (rawagner) [reply frontend/public/components/utils/units.js]: eslint has a different opinion so I cannot add braces :)

Rebased

Comment (vojtechszocs) [reply frontend/public/components/utils/units.js]: Oh, ESLint :smiley

**Comment by jelkosz** [comment]
Just rebased, adding lgtm back.

/lgtm

**Pull request merged**